### PR TITLE
Update running-iis-express-from-the-command-line.md

### PR DESCRIPTION
**Pull request by Trace-Elements0**
Adding a note for powershell users who may attempt to use powershell's ability to run cmd commands

### CODE REVIEW

File: iis/extensions/using-iis-express/running-iis-express-from-the-command-line.md [hunk 1]
```diff
@@ -38,6 +38,9 @@ For information about how to download and install IIS Express, see [IIS Express
 3. Run the following command to view the IIS Express usage string:
 
     [!code-console[Main](running-iis-express-from-the-command-line/samples/sample3.cmd)]
+         
+     > [!NOTE]
+     > If you're using PowerShell, `iisexpress /?` will not work. To instruct PowerShell to run the command in the current directory you must prefix the command with `.\`. So the first command in step 3 becomes `.\iisexpress /?`
```
Comment (Rick-Anderson) [inline iis/extensions/using-iis-express/running-iis-express-from-the-command-line.md:43]: In PowerShell, the first command is `.\iisexpress /?`

Comment (Rick-Anderson) [reply iis/extensions/using-iis-express/running-iis-express-from-the-command-line.md:43]: @Trace-Elements0  you've disabled writing to your branch. Can you replace your note with my version?

**Review by Rick-Anderson — CHANGES_REQUESTED** [review]

**Comment by Rick-Anderson** [comment]
@Trace-Elements0  can you resubmit this PR and not block commits?

**Comment by Trace-Elements0** [comment]
Will do. Not sure how I blocked commits, sorry about that.

**Comment by Rick-Anderson** [comment]
> Will do. Not sure how I blocked commits, sorry about that.

There is a check box you must of accidently unchecked. No problem.

**Pull request closed without merging**